src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.4.0
```

Comment:
We discussed a bit yesterday about the readability of our changelogs... does it make sense point people to read further down and review all the changes from all the RCs?

```suggestion
## 1.4.0

Released 2023-Feb-24. This is the stable release encompassing work from 1.4.0-alpha.1 to 1.4.0-rc.4
```

Alternatively we could just do this in the GitHub release log.

Reply:
I would say it's better to do it in the GitHub release log. I would expect people who look at the CHANGELOG to know that the changes are added incrementally.

Reply:
Either seems good by me.